lifecycle/lifecycle/job/models_registry.py

```diff
@@ -89,7 +90,8 @@ def read_latest_wildcard_job_model(job_name: str, version_wildcard: str) -> mode
     """
     version_pattern = SemanticVersionPattern.from_x_pattern(version_wildcard)
 
-    job_queryset = models.Job.objects.filter(name=job_name)
+    job_queryset = models.Job.objects.filter(name=job_name, status__in=[
```

Comment:
Why the double underscore for `status__in`?

Reply:
Good question. That's the way of applying filters in Django. Perhaps to distinguish field name from an operator. https://docs.djangoproject.com/en/5.0/topics/db/queries/#field-lookups
https://docs.djangoproject.com/en/4.2/ref/models/querysets/#in

> Basic lookups keyword arguments take the form field__lookuptype=value. (That’s a double-underscore)